target/linux/rockchip/armv8/base-files/etc/board.d/01_leds

```diff
@@ -9,6 +9,9 @@ boardname="${board##*,}"
 board_config_update
 
 case $board in
+friendlyarm,nanopi-neo3)
+	ucidef_set_led_netdev "lan" "STAT" "$boardname:green:sys" "eth0"
```

Comment:
So, is it "lan", "stat" or "sys"?

Reply:
Since this appears to have lan _and_ wan LEDs, one should use these appropriately.

Reply:
> So, is it "lan", "stat" or "sys"?

Howerver, there's only a LED on NEO3. In accordance with practice, maybe
it's more fair to set it as "sys."

Reply:
And why do you add three LEDs in your kernel patch then?

Reply:
> And why do you add three LEDs in your kernel patch then?

This patch is outdated and just needs be updated by author.
Also, he had send it to kernel, just ping him and see what's new.

Reply:
Well, this is your submission, so it needs to be updated by you?

Reply:
> Well, this is your submission, so it needs to be updated by you?

Nope, this was actually created by @mj22226.

@mj22226 Could you please have a look? or is it still in wip?

Reply:
Well, somebody will have to make it proper, it's not so important who that will be ...

Reply:
As I don't have such a device, I'm sorry for that I cannot really take a look on it.

Reply:
@1715173329 Sorry, I was persistently under the impression that you were the author. Sorry for my tone and insistence and thanks for providing the documentation.

Reply:
@adschm Never mind, please.
Making codes better is what we always do, so yes, I just saw a conscientious and responsible
developer, but nothing else.

Also, I'm glad if I could do anything for it :-)